README.md

```diff
@@ -163,6 +163,72 @@ If you are running the service in Docker, you will also need to:
 1. Set the `OLLAMA_BASE_URL` environment variable to the base URL of the Ollama server, e.g. `OLLAMA_BASE_URL=http://host.docker.internal:11434`
 1. Alternatively, you can run `ollama/ollama` image in Docker and use a similar configuration (however it may be slower in some cases).
 
+### Using VertexAI
```

Comment:
This note is pretty long for the main readme, can we add a docs/ folder, put in a separate markdown file and just add a brief link here?

Reply:
@JoshuaC215 

Certainly.

Reply:
Alternatively, if Google has an existing doc page for this with the same information, you could just link to that. Most of this doesn't seem too specific to the repo.

Otherwise looks good to merge after those changes. thanks for the contribution!

Reply:
@JoshuaC215 

Google does have links (some in the readme) but I just wanted to outline the process a little more since I did not feel it was completely obvious when getting started.   Perhaps I went a little overboard.